src/batched/KokkosBatched_InnerTrsm_Serial_Impl.hpp

```diff
@@ -1366,7 +1366,7 @@ namespace KokkosBatched {
       inv_a_00 = static_cast<ValueType>(1.0)/A[0*_as0+0*_as1];
 
     auto trsv = [&](const int p,
-                    ValueType &b_0p) {
+                    ValueType &/* b_0p */) {
```

Comment:
We could just get rid of this unused argument, since doing so would not break backwards compatibility.  The unused nonconst reference argument might even interfere with unrolling (see `#pragma unroll` below).

Reply:
Not sure if you want to keep it in here because `/* b_0p /= a_00; */` is commented out below&mdash;maybe that will get uncommented at some point?  I don't care either way.

Reply:
@jmgate It's cool, don't worry :)